crates/iota-network/src/randomness/tests.rs

```diff
@@ -473,10 +490,7 @@ async fn test_byzantine_peer_handling() {
     }
     for rx in &mut randomness_rxs[..2] {
         // Validators (0, 1) can communicate normally in new epoch.
-        let (epoch, round, bytes) = rx.recv().await.unwrap();
-        assert_eq!(1, epoch);
-        assert_eq!(0, round.0);
-        assert_ne!(0, bytes.len());
+        receive_with_timeout(rx, 1, 0).await.unwrap();
```

Comment:
maybe use expect here.

Reply:
Fixed in https://github.com/iotaledger/iota/pull/5732/commits/cc484523a02668f5aad3d643119797be9e10a989.